src/web/components/App.tsx

```diff
@@ -869,6 +881,7 @@ export const App = ({ CAPI, NAV }: Props) => {
 					isPaidContent={CAPI.pageType.isPaidContent}
 					tags={CAPI.tags}
 					contributionsServiceUrl={CAPI.contributionsServiceUrl}
+					brazeMessages={brazeMessages}
```

Comment:
We don't have the concept of, er, `PortalOnce` so these will be rerendered. I think we came to the conclusion that this is fine and that it should work like a normal react rerender.

The down side is everything in the file gets rerendered so we might want to think about memoisation at some point (which is what our ThingOnce pattern is essentially doing). Of course, sometimes you _want_ the rerender which is the case here.

Reply:
Good point. I _think_ this is fine, in both cases we use `useOnce` around the heavy lifting of figuring out if there's a message to show.